assets/src/edit-story/components/canvas/pagenav/index.js

```diff
@@ -68,6 +58,9 @@ function PageNav({ isNext }) {
     })
   );
   const { isRTL } = useConfig();
+
+  // Cancel lasso on mouse down
+  const cancelMouseDown = useCallback((evt) => evt.stopPropagation(), []);
```

Comment:
Tried to test on `main` to understand what was happening before but wasn't sure. Curious what was the previous behavior that this prevents now for the user.

Reply:
It was also there [before](https://github.com/google/web-stories-wp/pull/6272/files/ac66669804e66fce95eeda6eddbfe9059ee26739#diff-02139befcc819b7d91d045078dd199c489e48c9dce377d439faf7b0dbcdc8c52L90) - I just memoized it.